bundles/org.openhab.ui.homebuilder/web/i18n/zh-CN.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
@ThomDietrich This does not look okay to me I would say this is another language?

Reply:
I was in the process of setting up crowdin with the right configuration, should be okay now.